src/main/java/org/motechproject/newebodac/service/CampaignMessageService.java

```diff
@@ -0,0 +1,44 @@
+package org.motechproject.newebodac.service;
+
+import java.util.List;
+import java.util.UUID;
+import org.motechproject.newebodac.dto.CampaignMessageDto;
+import org.motechproject.newebodac.mapper.CampaignMessageMapper;
+import org.motechproject.newebodac.repository.CampaignMessageRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class CampaignMessageService {
+
+  @Autowired
+  private CampaignMessageRepository campaignMessageRepository;
+
+  private static final CampaignMessageMapper CAMPAIGN_MESSAGE_MAPPER =
+      CampaignMessageMapper.INSTANCE;
+
+  /**
+   * Returns all campaign messages (dtos) from the database.
+   * @return List of campaign message dtos.
+   */
+  public List<CampaignMessageDto> getCampaignMessagesDtos() {
```

Comment:
I don't think we will need the entity name in the method name, it can be just getAll()
change this in all services (remove the Dto from the name too)